src/pfe/file-watcher/server/test/functional-test/configs/pfe.config.ts

```diff
@@ -10,13 +10,16 @@
  *******************************************************************************/
 
 export const pfe = {
-    "PROTOCOL": process.env.PFE_PROTOCOL || "http",
+    "PROTOCOL": process.env.PFE_PROTOCOL || process.env.IN_K8 ? "https" : "http",
     "HOST": process.env.PFE_HOST || "localhost",
-    "PORT": process.env.PFE_PORT || "9090",
+    "PORT": process.env.PFE_PORT || process.env.IN_K8 ? "9191" : "9090",
 };
 
 const apis = ["/api/v1/"];
```

Comment:
I think we can name `apiv1` in case we have upper version API in the future. In Microclimate, some test cases use different version API, if we specify the version here, it's easier for us to apply API version to the following APIs for future.

Reply:
Sure, I can change it to call `apiV1`

Reply:
Do we need to call it `apiv1`? It is an array of various versions...so if we do have v2 api in the future, we can just add it to the array and it should be fine.

Reply:
Yes, it's an array, but for adding v2 we have to do this:
```
const apis = ["/api/v1/", "/api/v2/"];

export const pfeAPIs = {
    "projects": apis[0] + "projects",
    "registry": apis[1] + "registry"
};
```

Instead, I mean we can make it more readable for adding v2 for future like this:
```
const apiv1 = "/api/v1/";
const apiv2 = "/api/v2/";

export const pfeAPIs = {
    "projects": apiv1 + "projects",
    "registry": apiv2 + "registry"
};
```

Reply:
Sure, we can do that.